Update feature-request.md (#12873)
elastic/logstash: .github/ISSUE_TEMPLATE/feature-request.md

---
name: Feature Request

> about: Request a new feature or an enhancement of a current one

Reviewer: ```suggestion
about: Request a new feature or suggest an enhancement to a existing one
```

> about: Request a new feature or suggest an enhancement to a existing one

Reviewer 2: ```suggestion
about: Request a new feature or suggest an enhancement to an existing one
```
